.travis.yml

```diff
@@ -3,10 +3,8 @@ sudo: false
 language: java
 matrix:
   include:
-  - jdk: oraclejdk8
-    env: TRIPLEA_RELEASE=true
   - jdk: openjdk11
-    env: TRIPLEA_RELEASE=false
+    env: TRIPLEA_RELEASE=true
```

Comment:
Just was reminded by this:
It might be a good idea to keep a second build with java 12 if travis supports it.
Just so that we can see if any change we make in the future will block us from going forward even further.
This way the TRIPLEA_RELEASE env variable wouldn't look redundant.

Reply:
I feel like I could almost argue it both ways. Does not being able to build on java12 mean we won't be able to build on future versions? If we then never release a java12 version and jump, will the java12 builds have been of any value? Then again, the Java11 upgrade was pretty smooth, but was that because java 11 is mostly backward compatible, or was it because we were building with java11?

It looks like the JDK12 build is available under "openjdk-ea": https://sormuras.github.io/blog/2018-05-31-jdk-matrix.html

Adding to JDK12 does not seem like a good change to lump in with what is already included here. I'd also be hesitant to remove the flag, while it would not be necessary, and if long-term was completely unused, removing it now seems like over eager/premature YAGNI

Reply:
I think JDK 12 is a bad example for this, because almost no features were added in this version.
Back when Java 9 was new, the first time I tried to build triplea I got a couple of compile errors due to some issues with generics and casting. (Java 9 introduced generics for some legacy classes IIRC, which broke compile-time functionality).
Of course this issue was already present at that point, but if this code would've been introduced when there was a java 9 build in place, it could've been avoided there.
Also as I mentioned already the Java 11 build (because of the JavaFX stuff) ensured that Java 11 would work flawlessly after the merge, if it wasn't in place, because of the weird dependency stuff we'd have to figure out what was going on first before being able to make this PR.

Reply:
But yeah, adding it in in a future PR is also fine by me.